unison-cli/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1140,6 +1134,19 @@ loop e = do
               Cli.respond $ SlurpOutput input (PPE.suffixifiedPPE ppe) sr
               addDefaultMetadata adds
               syncRoot description
+            SaveExecuteResultI resultName -> do
+              description <- inputDescription input
+              let resultVar = Name.toVar resultName
+              uf <- addSavedTermToUnisonFile resultName
+              Cli.Env {codebase} <- ask
+              currentPath <- Cli.getCurrentPath
+              currentNames <- Branch.toNames <$> Cli.getCurrentBranch0
+              let sr = Slurp.slurpFile uf (Set.singleton resultVar) Slurp.AddOp currentNames
+              let adds = SlurpResult.adds sr
```

Comment:
What if the user provides a name that already exists in the codebase? 
I think we need to handle updates here too, it's likely users will want to update definitions by running `main` again in the future.